_posts/2023-11-09-platform-protections.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+date: November 9th 2023
+title: "New platform protections against malicious activity"
+---
+
+# New platform protections against malicious activity!
+
+In response to some [recent incidents that caused outages on the platform](https://cloudgov.statuspage.io/incidents/n212qfbrqg83), the cloud.gov team has added new [platform protections against malicious activity](https://cloud.gov/docs/technology/platform-protections/). Notably, the team has added rate limiting by IP address to mitigate the effect of surges of malicious traffic on the platform.
```

Comment:
```suggestion
In response to some [recent incidents that caused outages on the platform](https://cloudgov.statuspage.io/incidents/n212qfbrqg83), the cloud.gov team has added new [platform protections against malicious activity]({{ site.baseurl }}{% link _docs/technology/platform-protections.md %}). Notably, the team has added rate limiting by IP address to mitigate the effect of surges of malicious traffic on the platform.
```

We need to use the Jekyll syntax for generating links in case our domain ever changes from `cloud.gov` to something else. Unlikely, but possible